frontend/src/app.vue

```diff
@@ -189,7 +213,7 @@ const app = {
       window.encodeHash();
     },
 
-    renderAuthorShipTabHash(minDate, maxDate) {
+    renderAuthorShipTabHash(minDate: string, maxDate: string) {
```

Comment:
```suggestion
    renderAuthorshipTabHash(minDate: string, maxDate: string) {
```
For the 'render X tab hash' functions, maybe we could have interfaces for the `info`/`zoomInfo` we are going to store?

Reply:
Good point! Just added in the types for `zoomInfo` and `authorshipInfo` - also noticed that certain fields are not added on refresh, but added to the objects later - made these fields optional in the interface declarations, and we can assert the existence of these fields in `zoomView` and `authorshipView` when the fields are accessed (when we migrate those files over). Thanks!